pantos/client/library/api.py

```diff
@@ -123,10 +126,13 @@ def retrieve_token_balance(blockchain: Blockchain, account_id: _AccountId,
         account.
     token_id : BlockchainAddress or TokenSymbol
         The address or symbol of the token (default: PAN).
-    return_in_main_unit : bool
+    return_in_main_unit : bool, optional
```

Comment:
This parameter is not new, I just added the missing "optional" in the docstring. However, the client library provides more than just a raw interface to contract functions. This one was intended for developer convenience (e.g. return 1 PAN as `1.0` instead of `100000000` by automatically accounting for a token's decimals). This is heavily used by the CLI, for example.